docs/changes/coordinates/13484.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Deprecate and remove internal usage of the ``BaseFrame.get_frame_attr_names``
+method, which actually returns an expensive to build up to now returned dict of names pointing to their defaults.
+If you need the defaults, use the new ``BaseFrame.get_frame_attr_defaults`` instead.
```

Comment:
Did something go wrong during the rewrite? And there is no need to mention that internal usage has changed, that is an implementation detail the end users should not care about.

Reply:
yes, fixed